source/core/parameters.cc

```diff
@@ -514,15 +514,20 @@ namespace Parameters
   }
 
   void
-  SurfaceTensionParameters::parse_parameters(ParameterHandler &prm)
+  SurfaceTensionParameters::parse_parameters(
+    ParameterHandler                &prm,
+    const Parameters::Dimensionality dimensions)
   {
     surface_tension_coefficient = prm.get_double("surface tension coefficient");
-    T_0                         = prm.get_double("reference state temperature");
+    surface_tension_coefficient *= dimensions.surface_tension_scaling;
+    T_0 = prm.get_double("reference state temperature");
```

Comment:
T_0 should be scale also I think

Reply:
Why aren't the dimensions multiplied in the same line?